xonsh/main.py

```diff
@@ -446,8 +442,10 @@ def func_sig_ttin_ttou(n, f):
             # enter the shell
             env["XONSH_INTERACTIVE"] = True
             ignore_sigtstp()
-            if env["XONSH_INTERACTIVE"] and not any(
-                os.path.isfile(i) for i in env["XONSHRC"]
+            if (
+                env["XONSH_INTERACTIVE"]
+                and not args.norc
```

Comment:
I think that the welcome screen should print `--no-rc` was passed in.  With this option xonsh, should function as if you were in a clean environment.

Reply:
That is how I have thought about `--no-rc` but am open to other opinions.  Thoughts @xonsh/xore?

Reply:
I understand the logic and revert this change. Please no discuss in this micro changes...